src/resources/views/fields/upload.blade.php

```diff
@@ -8,7 +8,7 @@
     <div class="well well-sm">
         @if (isset($field['disk']))
         @if (isset($field['temporary']))
-            <a target="_blank" href="{{ (asset(\Storage::disk($field['disk'])->temporaryUrl(array_get($field, 'prefix', '').$field['value'], Carbon\Carbon::now()->addMinutes($field['temporary'])))) }}">
+            <a target="_blank" href="{{ (asset(\Storage::disk($field['disk'])->temporaryUrl(array_get($field, 'prefix', '').$field['value'], Carbon\CarbonInterface::now()->addMinutes($field['temporary'])))) }}">
```

Comment:
Interfaces are just contracts. All its methods are empty that classes which `implements` the interface need to have.

So you should use Carbon\Carbon::now() or Carbon\CarbonImmutable::now() here instead. CarbonImmutable if you want to be safe from the side effects of mutability which I don't see any here yet.

Reply:
You're totally right. I changed a bunch of them back, in columns revisions etc. Seems to have missed this one. Thanks!